### ISSUE TITLE
Overall: How to reduce ISO size

### ISSUE THREAD

**Issue by star-buck** [blue-systems/netrunner-core-releases#22]

**Comment by llelectronics** [comment]
Here a list of packages and their sizes on 19.01 core currently:
[list_packages_core_1901.txt](https://github.com/blue-systems/netrunner-core-releases/files/2768239/list_packages_core_1901.txt)

**Comment by llelectronics** [comment]
From that list one can see libicu60 we can get rid off. Nodejs is apparently installing it as dependency. However something might recommend nodejs I am not sure currently which package though. So it removes nodejs and some other packages. All of them not necessary for Core imho
![image](https://user-images.githubusercontent.com/3415572/51312980-c8c98f00-1a4c-11e9-9137-b263f52cbb3d.png)
Same goes for freepats

**Comment by zgrimshell** [comment]
apt rdeps nodejs should help here

**Comment by llelectronics** [comment]
You mean apt rdepends nodejs. Yeah not really it gives a long long list. Anyway I added some of the packages to the be removed by live-build. See: https://github.com/netrunner-core/live-build/commit/1bebdd3aa664588c3a4db073c77ff02b05cca822

**Comment by zgrimshell** [comment]
yes, nodejs does have a long list but that is how you do the check (you could run one file against other to check what they have in common)

**Comment by zgrimshell** [comment]
checked the list, good that nodejs is going out from default :)

**Comment by hsitter** [comment]
> Here a list of packages and their sizes on 19.01 core currently:
> [list_packages_core_1901.txt](https://github.com/blue-systems/netrunner-core-releases/files/2768239/list_packages_core_1901.txt)

I think you need to get a list of compressed package sizes. Given the squashfs of the ISO is hopefully compressed (@shadeslayer is it?) the uncompressed size of packages means nothing. Case in point: libgl1-mesa-dri compressed should be like 6mib

**Comment by shadeslayer** [comment]
It's compressed.

**Comment by llelectronics** [comment]
Removing some of the packages like stated before resulted in a slight improvement:
`-rw-r--r--  1 leszek leszek 1739718656 Jan 16 18:07 netrunner-core-20190116.1638-amd64.hybrid.iso`
`-rw-r--r--  1 leszek leszek 1677721600 Jan 17 14:12 netrunner-core-20190117.1201-amd64.hybrid.iso`
